openedx/core/djangoapps/content/learning_sequences/migrations/0002_auto_20200721_1224.py

```diff
@@ -0,0 +1,65 @@
+# Generated by Django 2.2.14 on 2020-07-21 12:24
```

Comment:
@ahmedbilal, can you use a meaningful name for the migration instead of using an auto-generated one? Also, please try to create migrations in logically coherent groups, combining/splitting migrations in this PR as needed. Can you combine this and third migration or split it into logical groups?